images/covreport/coverage-report.sh

```
@@ -0,0 +1,137 @@
+fi
+
+PR_COVERAGE=$(extract_total_coverage "${ARTIFACTS}/pr.cov")
+echo "PR coverage: ${PR_COVERAGE}%"
+
+# Run coverage on base branch
+BASE_COVERAGE="0"
+CURRENT_HEAD=$(git rev-parse HEAD)
+
+git fetch origin "${PULL_BASE_SHA}" --depth=1 2>/dev/null || true
+if git checkout "${PULL_BASE_SHA}" 2>/dev/null; then
+    if go test ${TEST_PACKAGES} -coverprofile="${ARTIFACTS}/base.cov" 2>/dev/null; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)